src/helpers/hooks/videoPlayer.ts

```diff
@@ -0,0 +1,39 @@
+import Hls from "hls.js";
+import { RefObject } from "react";
+
+export const handleSavedTime = (
+  videoElement: HTMLVideoElement,
+  videoId: string
+): void => {
+  const savedTime = localStorage.getItem(videoId);
+
+  if (savedTime !== null) {
```

Comment:
as we are using TypeScript, you can do `if(savedTime)` instead `if (savedTime !== null)`. It is equal conditions. If it`s not, reply on this comment please.

Reply:
I fix it in hw-4, but forget add here